SurveyApp/Extensions/UIKit/UIImage/UIImage+Creation.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIImage+Creation.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 7/7/21.
+//
+
+import UIKit
+
+extension UIImage {
+
+    static func from(color: UIColor, size: CGSize = CGSize(width: 1, height: 1)) -> UIImage? {
+        let rect = CGRect(origin: CGPoint(x: 0, y: 0), size: size)
+        UIGraphicsBeginImageContext(size)
+        let context = UIGraphicsGetCurrentContext()
+        context?.setFillColor(color.cgColor)
+        context?.fill(rect)
+        let image = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        return image
+    }
+}
```

Comment:
```suggestion
extension UIImage {

    static func withBackground(color: UIColor, size: CGSize = CGSize(width: 1, height: 1)) -> UIImage? {
        let renderer = UIGraphicsImageRenderer(size: size)
        return renderer.image { ctx in
            
    color.setFill()
    ctx.fill(CGRect(x: 0, y: 0, width: size.width, height: size.height))
            self.drawHierarchy(in: self.bounds, afterScreenUpdates: true)
        }
    }
}
```

Please use `UIGraphicsImageRenderer` as it is faster https://medium.com/the-traveled-ios-developers-guide/uigraphicsimagerenderer-fe40edc3a464

Please check if this code is working as expected or not.